server/eclipse-project/src/main/java/us/freeandfair/corla/csv/ColoradoBallotManifestParser.java

```diff
@@ -207,20 +228,14 @@ public synchronized boolean parse() {
     my_parse_status = true;
     my_parse_success = result;
     
-    // if we had any kind of parse error, we scrap the whole import
-    
-    if (!result) {
```

Comment:
What is now reponsible for the equivalent of `Persistence.delete($Class, $id)`, or how is it sidestepped? I see that below you were able to remove the other `abort()` methods, too.